### PR TITLE
[Debugger] Add some info about substitution font

### PR DESCRIPTION
**Pull request by calixteman**
When pdfBug is true, the substitution font is used in the text layer in order to be able to know what is the font really used thanks to the devtools. And to be sure that fonts are loaded, the font cache isn't cleaned up when the debugger is active.

### CODE REVIEW

File: web/app.js [hunk 1]
```diff
@@ -1816,7 +1816,7 @@ const PDFViewerApplication = {
     this.pdfViewer.cleanup();
     this.pdfThumbnailViewer?.cleanup();
 
-    this.pdfDocument.cleanup();
+    this.pdfDocument.cleanup(/* keepLoadedFonts = */ AppOptions.get("pdfBug"));
```
Comment (Snuffleupagus) [inline web/app.js:1819]: I'm not sure that I understand, more specifically, why this is necessary for *non*-embedded fonts? I.e. what potentially "breaks" in the `Font Inspector` unless we keep fonts around?

But, assuming that this is in fact necessary:
```suggestion
    this.pdfDocument.cleanup(/* keepLoadedFonts = */ AppOptions.get("fontExtraProperties"));
```

**Review by Snuffleupagus — CHANGES_REQUESTED** [review]

File: web/app.js (hunk 1 above)
Comment (calixteman) [reply web/app.js:1819]: In the devtools, in the font panel, we've some infos about the font used. If we clear the cache then we'll see the font used as ultimate fallback (serif/sans serif).

**Review by Snuffleupagus — APPROVED** [review]
r=me, thank you!

**Pull request merged**